src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -180,14 +181,18 @@ public void NonAppJsonTypeServiceDefinitionForConsumes()
         [Fact]
         public void NonHttpsServiceDefinitionForScheme()
         {
-            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "non-https-service-def-scheme.json"));
+            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "non-https-service-def-scheme.json"),
+                                           ServiceDefinitionDocumentType.ARM,
+                                           ServiceDefinitionMergeState.Before);
```

Comment:
referring to my previous comment here, when I read ARM, I could guess this is ARM related rules, but reading MergeState.Before doesn't tell me much